libs/infinity_emb/infinity_emb/infinity_server.py

```diff
@@ -220,13 +225,13 @@ async def _embeddings(data: OpenAIEmbeddingInput):
             if isinstance(data.input, str):
                 data.input = [data.input]
 
-            logger.debug("[📝] Received request with %s inputs ", len(data.input))
+            logger.info("[📝] Received request with %s inputs ", len(data.input))
```

Comment:
Do we need the icon in logging?

Reply:
I have kept it unchanged from upstream. Do you foresee any issues due to this?